src/scripts/dot-gov.js

```diff
@@ -0,0 +1,363 @@
+/** Description:
+ * Be entertained (and maybe helped) with .gov domains.
+ *
+ * In the git.gov version, Charlie listens for "git.gov" in any message
+ * and responds with a random government domain.
+ *
+ * In the get .gov version, Charlie must be addressed directly. This
+ * version allows filtering by domain type and searching.
+ *
+ * The format is @Charlie [entity] <trigger> [search term].
+ *
+ * If an entity is specified, Charlie limits the search to domains
+ * of that entity type.
+ *
+ * In the case that a search term is given, Charlie responds
+ * with up to five randomly chosen matches or with a message
+ * indicating that no domain matched the search.
+ *
+ * In the case of no search term, Charlie responds with one
+ * random domain.
+ *
+ *  Dependencies:
+ *    "csv-parse"
+ *
+ * Commands:
+ *    (entity) get .gov (search term)
+ *
+ *    examples:
+ *        git.gov
+ *        @Charlie get .gov
+ *        @Charlie .gov
+ *        @Charlie tribal .gov
+ *        @Charlie city .gov Lanesboro
+ */
+
+const { directMention } = require("@slack/bolt");
+const axios = require("axios");
+const https = require("https");
+
+/* eslint-disable import/no-unresolved */
+const { parse } = require("csv-parse/sync");
+/* eslint-enable import/no-unresolved */
+
+const { cache, helpMessage } = require("../utils");
+
+/** A regex string for searching the 9 types of .gov domain entities in a variety of ways. */
+const domainTypesRegex = [
+  "Gov(ernment)?",
+  "City",
+  "County",
+  "Executive(\\sBranch)?",
+  "Judicial(\\sBranch)?",
+  "Legislative(\\sBranch)?",
+  "Fed(eral)?",
+  "(Ind(ependent)?\\s)?Intra(state)?",
+  "Inter(state)?",
+  "State",
+  "Trib(e|(al))",
+]
+  .map((domain) => `(${domain})`) // add parentheses around each group
+  .join("|"); // join them with pipe separators
+
+/** A regex string for catching the user's trigger. */
+const gitGovRegex = new RegExp(
+  [
+    `((?<re_type>${domainTypesRegex})\\s)?`,
+    "((get\\s?)?\\.gov)",
```

Comment:
Actually, scratch this. The word boundary wouldn't fix it and I'm genuinely not sure why I thought it would. 😅

Reply:
> I believe this line is why `@charlie git.gov` triggers twice. It will match anything that has `.gov` in a directed message to Charlie. E.g., `@charlie bob.gov` will match.

I think you are probably right. I'm going to leave it as-is, because if I make changes to the regex I'm not going to feel comfortable merging until I've done another round of manual testing, which I do not have time to do. :sad-panda: